mopro-ffi/src/mopro.udl

```diff
@@ -14,6 +14,13 @@ namespace mopro {
   [Throws=MoproError]
   boolean verify_proof2(bytes proof, bytes public_input);
 
+[Throws=MoproError]
```

Comment:
(Indent)

Later on we could consider splitting this udl file up depending on flag, but I think this is fine for now!